test/core/tsi/s2a/BUILD

```diff
@@ -21,7 +21,9 @@ grpc_package(name = "test/core/tsi/s2a")
 grpc_cc_library(
     name = "s2a_test_util",
     srcs = ["s2a_test_util.cc"],
-    hdrs = ["s2a_test_util.h"],
+    hdrs = [
```

Comment:
Is this auto-generated? If not please revert this change to keep the PR simple, unless it was inherited from an upstream merge.

Reply:
This was auto-generated. Not sure why it makes this change sometimes and leaves it others.